src/test.rs

```diff
@@ -82,24 +82,22 @@ fn get_system_info() {
 #[cfg(test)]
 fn get_images() {
     let response = get_images_response();
-    let _: Vec<Image> = match json::decode(&response) {
-        Ok(body) => body,
-        Err(_) => { assert!(false); return; }
-    };
```

Comment:
Good cleanup. :-) When you otherwise have a good reason to touch a function, it's totally OK to fix any weird style.